common/Utils.hpp

```diff
@@ -13,6 +13,13 @@
 #include <typeinfo>
 #include <vector>
 #include <QtWidgets>
+#include <memory>
+namespace std {
```

Comment:
But this way it's easy replaceable when we finally upgrade to c++14. None of the code needs to change except that we delete this function.